src/app/forms/templates/test-records/create-master.template.ts

```diff
@@ -492,8 +492,8 @@ export const contingencyTestTemplates: Record<VehicleTypes, Partial<Record<keyof
       required: CreateRequiredSectionLgvCar,
       vehicle: ContingencyVehicleSectionDefaultPsvHgvLight,
       test: ContingencyTestSectionSpecialistGroup1,
-      defects: DefectsTpl,
-      customDefects: CustomDefectsHiddenSection,
+      defects: defectsHiddenSection,
```

Comment:
do we not want to remove the hidden section? I thought we changed the backend so it was no longer required, and leaving it in caused some bugs for the result of test calculation?

Reply:
This needed rebasing from develop.